### PR TITLE
Add estimate for influenza

### PR DESCRIPTION
**Pull request by jeffkaufman**
This follows the same pattern as Norovirus for estimating the short-term incidence of something seasonal: scale an overall estimate that should be reasonably accurate by a more detailed estimate that isn't calibrated.

### CODE REVIEW

File: pathogen_properties.py [hunk 1]
```diff
@@ -173,7 +173,11 @@ def summarize_location(self, all_locations=None):
 
         return "; ".join(
             sorted(
-                set(i._location() for i in self.all_inputs if i._location())
+                set(
+                    i._location()
+                    for i in self.all_inputs | set([self])
```
Comment (jeffkaufman) [inline pathogen_properties.py:178]: While I was in here I noticed and fixed a bug where summarize_location and summarize_date were missing that location/date information could be on the outer variable

**Review by asherps — APPROVED** [review]
This looks great as far as I can tell

File: pathogens/influenza.py [hunk 2]
```diff
@@ -0,0 +1,260 @@
...
+                int(positive_a),
+                int(positive_b),
+            )
+    return output
+
+
+# There's a ton of studies on this, but they vary a lot.  Fielding 2013 has a
+# metaanalysis but includes "Given the significant heterogeneity in most
+# groups, the combined estimates of viral shedding duration are not reported."
+shedding_duration = SheddingDuration(
```
Comment (simonleandergrimm) [inline pathogens/influenza.py:114]: This isn't fecal shedding, right? If so I would note that. You can also check this paper for more information on shedding of flu in stool: https://twist.com/a/197793/ch/570556/t/4390298/

File: pathogens/influenza.py [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+    # From eyeballing the "Community" portion of Figure 1.
+    days=5,
+    source="https://onlinelibrary.wiley.com/doi/full/10.1111/irv.12216",
+)
+
+infections_2019_2020 = IncidenceAbsolute(
```
Comment (simonleandergrimm) [inline pathogens/influenza.py:120]: There is an uncertainty in the table below: https://www.cdc.gov/flu/about/burden/2019-2020.html#:~:text=All%20ages,(28%2C663%2C846%2C%0A71%2C197%2C342)

File: pathogens/influenza.py [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+    annual_infections=36_000_000,
+    start_date="2019-07-01",
+    end_date="2020-07-01",
+    tag="us-2019-2020",
+    # "The overall burden of influenza (flu) for the 2019-2020 was an estimated
+    # 36 million flu-related illnesses"
+    source="https://www.cdc.gov/flu/about/burden/2019-2020.html#:~:text=The%20overall%20burden%20of%20influenza%20(flu)%20for%20the%202019%2D2020%20was%20an%20estimated%C2%A036%20million%C2%A0flu%2Drelated%20illnesses",
+)
+
+infections_2021_2022 = IncidenceAbsolute(
```
Comment (simonleandergrimm) [inline pathogens/influenza.py:130]: ditto: https://www.cdc.gov/flu/about/burden/2021-2022.htm#:~:text=All%20ages,(7%2C683%2C440%2C%2015%2C541%2C059)

File: pathogens/influenza.py [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+def estimate_prevalences() -> list[Prevalence]:
+    # State -> Week -> (PositiveA, PositiveB)
+    weekly_data = load_weekly_data()
+
+    # We can't just go from positive tests to prevalence because many people
+    # will get flu but never be tested, or their test won't make it back to the
+    # CDC.  For each flu season we use the overall CDC incidence estimate and
+    # our count of positive tests to get an estimate of underreporting.
+
+    underreporting_2019_2020 = compare_incidence_to_positive_tests(
```
Comment (simonleandergrimm) [inline pathogens/influenza.py:184]: Just so I understand this, is this underreporting scalar calculated by comparing overall positive tests in one year with the estimated number of cases in that same year?

**Review by simonleandergrimm — CHANGES_REQUESTED** [review]
Very nice work! See comments for feedback.

File: pathogens/influenza.py (hunk 2 above)
Comment (jeffkaufman) [reply pathogens/influenza.py:118]: added todo

File: pathogens/influenza.py (hunk 3 above)
Comment (jeffkaufman) [reply pathogens/influenza.py:124]: awesome, thanks!  added

File: pathogens/influenza.py (hunk 4 above)
Comment (jeffkaufman) [reply pathogens/influenza.py:136]: added

File: pathogens/influenza.py (hunk 5 above)
Comment (jeffkaufman) [reply pathogens/influenza.py:192]: yes

**Pull request merged**